data_processing.py

```diff
@@ -0,0 +1,128 @@
+import os
+import pandas as pd
+import torch
+from torch.utils.data import (DataLoader, RandomSampler, SequentialSampler, TensorDataset)
+from collections import namedtuple
+
+class LeftRightProcessor: 
+    """ The LeftRight Processor handles getting data from csv files to a format useful to the model. """
+    def __init__(self, max_seq_length, tokenizer, data_dir, seed, train_batch_size, eval_batch_size):
+        self.max_seq_length = max_seq_length
+        self.tokenizer = tokenizer
+        self.data_dir = data_dir
+        self.seed = seed
+        self.train_batch_size = train_batch_size
+        self.eval_batch_size = eval_batch_size
+        
+    def get_labels(self):
```

Comment:
Possibly this wants to be an attribute and not a function (self.labels = `["left", "right"]` in `__init__` e.g.). But if you anticipate the labels changing a lot, or involving conditional logic, a function is a fine place.